ignite/engine/state.py

```diff
@@ -33,21 +35,22 @@ class State:
         kwargs: keyword arguments to be defined as State attributes.
     """
 
-    event_to_attr = {
```

Comment:
You have to keep it for BC as `event_to_attr` is a public class attribute. Try to see if we can deprecate it on its usage somehow (e.g. as property?)

Reply:
That raised the deprecation warning in case of just using it as an instance attribute, but I think that's not enough, I am trying to find a way to raise a deprecation warning while using it as a class attribute